source/globalCommands.py

```diff
@@ -3458,7 +3459,11 @@ def script_reportClipboardText(self,gesture):
 		else:
 			# Translators: If the number of characters on the clipboard is greater than about 1000, it reports this message and gives number of characters on the clipboard.
 			# Example output: The clipboard contains a large portion of text. It is 2300 characters long.
-			ui.message(_("The clipboard contains a large portion of text. It is %s characters long") % len(text))
+			ui.message(ngettext(
+				"The clipboard contains a large portion of text. It is %s character long",
+				"The clipboard contains a large portion of text. It is %s characters long",

```

Comment:
English grammar: should we write "It is 500 character long" or "It is 500 characters long"?

Personally, I would have not put a "s" to "character", but the original string had one.
```suggestion
				"The clipboard contains a large portion of text. It is %s character long",
```

Note that in any case, the use of `ngettext` is still needed because the translations have a distinct form for plural in this case.

Reply:
> Thus, it seems the original should be considered correct.

Thanks for the confirmation. Actually, I realize that I was making a confusion with another type of construction such as "It's a 500 character long string" where there would actually not be an S, "500 character long" considered as an adjective in this last example.